packages/twenty-front/src/modules/ui/input/editor/components/CustomSideMenuOptions.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { IconComponent } from 'twenty-ui';
+
+const StyledContainer = styled.div<{ variant: variants }>`
+  color: ${({ theme, variant }) =>
+    variant === 'Danger' ? theme.color.red : 'inherit'};
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
```

Comment:
For gap we usually use the spacing param, (`${({ theme }) => theme.spacing(2)};`)

Reply:
got it